tests/unit_tests/test_client.py

```diff
@@ -77,46 +96,105 @@ def cache(tmpdir):
 
 
 @pytest.fixture()
-def unleash_client(cache):
+def readyable_unleash_client(cache):
```

Comment:
Pytest fixures have some weird scope closing rules that makes just using the function above not possible. So each of these fixtures gets their own little copy of the code